scripts/bundlesize/bundlesize.config.json

```diff
@@ -2,7 +2,7 @@
     "files": [
         {
             "path": "packages/lwc/dist/engine-dom/umd/es2017/engine-dom.min.js",
-            "maxSize": "18.5KB",
+            "maxSize": "18.6KB",
```

Comment:
Feel free to bump it higher; I'm about to blow through this limit with pivots anyway. 😂